src/api/index.ts

```diff
@@ -126,10 +126,10 @@ export function fetchRegister<T = any>(username: string, password: string) {
 }
 
 export function fetchUpdateUserInfo<T = any>(name: string, avatar: string, description: string) {
-  return post<T>({
-    url: '/user-info',
-    data: { name, avatar, description },
-  })
+	return post<T>({
```

Comment:
So can you help check why eslint doesn't solve the indent format issue here?

Reply:
I'm sorry. It's all fixed now.